src/fairchem/core/models/base.py

```diff
@@ -8,27 +8,42 @@
 from __future__ import annotations
 
 import logging
+from abc import ABCMeta, abstractmethod
+from dataclasses import dataclass
+from typing import TYPE_CHECKING
 
 import torch
-import torch.nn as nn
+from torch import nn
 from torch_geometric.nn import radius_graph
 
+from fairchem.core.common.registry import registry
 from fairchem.core.common.utils import (
     compute_neighbors,
     get_pbc_distances,
     radius_graph_pbc,
 )
 
+if TYPE_CHECKING:
+    from torch_geometric.data import Batch
```

Comment:
nit: instead of torch_geometric.data, can we just declare something like type Batch = Union[Dict[torch.Tensor], torch.Tensor]?

Reply:
We just talked on chat, we should deal with torch_geometric.data.Batch in a new PR, where we fully remove it from out code base. I am opening a task for it